### ISSUE TITLE
GUI shield from unit missing

### ISSUE THREAD

**Issue by Loobinex** [OpenDungeons/OpenDungeons#1202]
I had a monk, and he did not have a green shield above his head. See the image below, the monk under the cursor.

![monkshield](https://cloud.githubusercontent.com/assets/13840686/15795617/9bf9290a-29f3-11e6-9d44-bd7bacb0b56f.jpg)

From git20160601

**Comment by hwoarangmy** [comment]
I've also already noticed that but I don't know where it comes from. It is not specific for the monk, I've seen it on many other creatures. But I don't think it will be easy to hunt.

**Comment by Loobinex** [comment]
Is there something you can add to logging related to these shields, so that if you see it happen we can provide a log file for this issue?

**Comment by hwoarangmy** [comment]
I don't think so because overlays are handled by ogre. But I will think about adding something to help debugging this. At least log overlay position computing when pressing a key to help see if the problem comes from there or from the ogre part.
